common/environment.go

```diff
@@ -47,6 +47,7 @@ var VisibleEnvironmentVariables = []EnvironmentVariable{
 	EEnvironmentVariable.CertificatePassword(),
 	EEnvironmentVariable.AutoTuneToCpu(),
 	EEnvironmentVariable.CacheProxyLookup(),
+	EEnvironmentVariable.UserAgentPrefix(),
```

Comment:
Are you sure we want this visible?  Previously I assumed you'd left it hidden on purpose, so that we would be less likely to get customers adding random prefixes that we are not interested in.

Reply:
Ze replied offline: sometimes people start integrating or embedding AzCopy without even telling us, so its good that this is discoverable for them.